src/Core/Addon/Module/ModuleManager.php

```diff
@@ -469,6 +482,8 @@ public function enable($name)
                 $e
             );
         }
+
+        $this->clearCache();
```

Comment:
I'm in doubt... how about clear the cache after the dispatch of the event? What will happens if a listener try to add a route or a new service for instance?

Reply:
in any case I'm afraid it won't work.. because even the clear cache is performed on process shutdown (I believe removing cache files during the process caused bugs)
So when the event is launched the service is not available yet in the container (nor the new routing)